apl-api/src/main/java/com/apollocurrency/aplwallet/api/dto/DexTradeInfoDto.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.api.dto;
+
+/**
+ *
+ * @author Serhiy Lymar
+ */
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.math.BigDecimal;
+
+@Schema(name="DexTradeInfoDto", description="Dex trade information")
+public class DexTradeInfoDto {
+
+    /**
+     * id for this entity is a database id.
+     */
+    @Schema(name="database id", description="Identificator of the entry insidt the database")
+    public long dbId;
+    @Schema(name="transaction ID", description="Identificator of exchange transaction")
+    public long transactionID;
+    @Schema(name="sender's offer ID", description="Identificator of exchange offer from the side of sender")
+    public long senderOfferID; 
+    @Schema(name="receiver's offer ID", description="Identificator of exchange offer from the side of receiver")
+    public long receiverOfferID;   
+    @Schema(name="Exchange type", description="buy/sell = 0/1")
+    public byte senderOfferType; 
+    @Schema(name="Type of currency", description="eth/pax/apl = 0/1/2")
+    public byte senderOfferCurrency; 
+    @Schema(name="Amount of currency for exchagne", description="parameter in gwei units")
```

Comment:
Typo `exchagne`

Reply:
As far as I'm concerned, there's no any Unit tests inside the apl-api project. Or at least, in no any neighboring classes are any tests available. I wish I could find any template for that purpose.

Reply:
As far as I'm concerned, there's no any Unit tests inside the apl-api project. Or at least, in no any neighboring classes are any tests available. I wish I could find any template for that purpose.